tests/runtests.py

```diff
@@ -58,15 +58,17 @@ def main() -> None:
         choices=("3.9", "3.10", "3.11", "3.12", "3.13"),
         help="Target Python version for the test (default: %(default)s).",
     )
-    parser.add_argument("path", help="Path of the stub to test in format <folder>/<stub>, from the root of the project.")
+    parser.add_argument(
+        "path", help=f"Path of the stub to test in format {PurePath('<folder>', '<stub>')}, from the root of the project."
```

Comment:
Same comment/question as below

```suggestion
        "path", help=f"Path of the stub to test in format {Path('<folder>', '<stub>')}, from the root of the project."
```

Reply:
I don't have enough knowledge to say whether Path or PurePath is better or if they are the same. PurePath lacks the logic that interacts with the filesystem, and in this case, the `<` and `>` characters are illegal in Windows. So, I would balk at using Path because I would be worried that Path might try to validate the "path" and throw an error. 

That same logic could be extended to suggest that `os.path.sep` is safer than `PurePath`.

Nevertheless, I don't know: I am speculating.

Path does work on my Windows system, though.
```pycon
(.venv) C:\apps\astToolkit>py
Python 3.13.3 (tags/v3.13.3:6280bb5, Apr  8 2025, 14:47:33) [MSC v.1943 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from pathlib import Path
>>> {Path('<folder>', '<stub>')}
{WindowsPath('<folder>/<stub>')}
>>> 
```